### PR TITLE
elf: recognize BPF_PROG_TYPE_SK_LOOKUP

### PR DESCRIPTION
**Pull request by lmb**
Add support for the new BPF socket lookup program and attach type.

### CODE REVIEW

**Comment by lmb** [comment]
cc @jsitnicki can you take a look?

File: types.go [hunk 1]
```diff
@@ -190,6 +167,15 @@ const (
 	AttachModifyReturn
 	AttachLSMMac
 	AttachTraceIter
+	AttachCgroupInet4GetPeername
+	AttachCgroupInet6GetPeername
+	AttachCgroupInet4GetSockname
+	AttachCgroupInet6GetSockname
+	AttachXDPDevmap
```
Comment (jsitnicki) [inline types.go:174]: I think 'dev map' is two words. Like 'CPU map'.

**Comment by jsitnicki** [comment]
Checked it against `include/uapi/linux/bpf.h` in bpf-next. LGTM.

File: types.go (hunk 1 above)
Comment (lmb) [reply types.go]: Done, thanks.

**Pull request merged**